docs/where-are-the-logs.md

```diff
@@ -8,5 +8,5 @@ Table below is scrollable, even if github's markdown renderer doesn't make that
 |-------------|---------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------|----------------------------------------------------------------------------------|
 | development |                                                                                                   | `code-dot-org/dashboard/test/ui/*.log`                                                                                               | in the console window where you ran `./up`                               | `code-dot-org/dashboard/log/development.log`                                           |
 | staging     | email<br/> To: `dev+build@code.org` <br/>Subject: `/home/ubuntu/staging/aws/build_and_mail_log websites`    |                                                                                                                                      | `ssh ubuntu@staging.code.org` `~/staging/code-dot-org/pegasus/log/*.log` | `ssh ubuntu@staging.code.org` `~/staging/code-dot-org/dashboard/log/staging.log` |
-| test        | email <br/> To: `dev+build@code.org` <br/> Subject: `/home/ubuntu/test/aws/build_and_mail_log test-websites`  | `ssh ubuntu@test.code.org`<br/>`~/test/code-dot-org/dashboard/test/ui/*.log`<br/>also on browserstack at https://www.browserstack.com/automate | `ssh ubuntu@test.code.org`<br/> `~/test/code-dot-org/pegasus/log/*.log`       | `ssh ubuntu@test.code.org` <br/>`~/test/code-dot-org/dashboard/log/test.log`          |
+| test        | email <br/> To: `dev+build@code.org` <br/> Subject: `/home/ubuntu/test/aws/build_and_mail_log test-websites`  | `ssh ubuntu@test.code.org`<br/>`~/test/code-dot-org/dashboard/test/ui/*.log`<br/>also on Sauce Labs at https://saucelabs.com/beta/dashboard/tests | `ssh ubuntu@test.code.org`<br/> `~/test/code-dot-org/pegasus/log/*.log`       | `ssh ubuntu@test.code.org` <br/>`~/test/code-dot-org/dashboard/log/test.log`          |
```

Comment:
I think an edit is correct here.  You'll no longer find (useful) UI test logs on browserstack, since they don't run there anymore.